bolt/outpost/settlement/v1/settlement.proto

```diff
@@ -272,6 +292,12 @@ message DepositBaseResponse {
   Asset amount = 1;
   // tx_id: The hash of the transaction.
   string tx_id = 2;
+  // epoch: Epoch to which balance was added, and in which new shares were minted.
+  uint64 epoch = 3;
```

Comment:
```suggestion
  optional uint64 epoch = 3;
```
I think this should be optional as this is not relevant in case of Cosmwasm outpost.

Reply:
CW Outpost will have its own SDK. The CW contracts never used any epoch-based architecture. 
I am not familiar with Sui contracts enough to know why they do though 😓